gamemodes/terrortown/gamemode/client/cl_equip.lua

```diff
@@ -979,6 +981,27 @@ function GM:OnContextMenuOpen()
 	end
 end
 
+-- Preload materials for the shop
+hook.Add("PostInitPostEntity", "TTTCacheEquipMaterials", function()
+	local itms = ShopEditor.GetEquipmentForRoleAll()
```

Comment:
I think we do not need to get the items from the `ShopEditor`, i guess we could get them from the `items` module.

Reply:
That's wrong, because Equipment isn't the same as an Item. An Equipment could be an Item AS WELL AS a weapon. Ref.: https://github.com/TTT-2/TTT2/blob/b7bfa4a03df2a2d000cf293a79587aafce7e3493/gamemodes/terrortown/gamemode/client/cl_shopeditor.lua#L72

Reply:
This should still be moved out of the shopeditor , when we rework its code